### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by bolshchikov**
Add issue and PR templates. 
Motivation:
This creates a common template for explaining the motivation of future changes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (electricmonk) [inline .github/ISSUE_TEMPLATE/bug_report.md]: please ask for log file rather than screenshot

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+If applicable, add screenshots of log files to help explain your problem.
+
+**Environment:**
+If applicable, add the information about that you are running and their versions
+
+
+**Additional context**
```
Comment (electricmonk) [inline .github/ISSUE_TEMPLATE/bug_report.md]: "if possible, attach a failing test"

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
```
Comment (electricmonk) [inline .github/ISSUE_TEMPLATE/feature_request.md]: "please consider adding a code example of how you'd like to use the feature"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
```
Comment (electricmonk) [inline .github/PULL_REQUEST_TEMPLATE.md]: "I have opened an issue and discussed the solution suggested in this PR"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+
+**Environment:**
+
+If applicable, add the information about that you are running and their versions
+
+
+**Additional context**
+
+If possible, provide the failing test the reveals the issue.
```
Comment (electricmonk) [inline .github/ISSUE_TEMPLATE/bug_report.md]: provide _a_ failing test; maybe link to a guide describing now to create an acceptance test?

Comment (bolshchikov) [reply .github/ISSUE_TEMPLATE/bug_report.md]: do you have such a guide?

Comment (electricmonk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: no, but we can create one :)

**Pull request merged**